edweiss-app/app/login/index.tsx

```diff
@@ -27,30 +51,114 @@ const Login: ApplicationRoute = () => {
 				router.replace("/");
 			} else {
 				console.log(accountRes.error);
-				setLoading(false);
+				setLoadingGoogle(false);
 			}
 		}
 	}
 
+	// Log the app Anonymously
+	async function signInAnonymous() {
+		setLoadingAnon(true);
+
+		const res = await signInAnonymously();
+
+		if (res != undefined) {
```

Comment:
I think you can simplify it by just if(res)

Reply:
Great suggestion! Yes, if (res) is enough here, as it checks for any truthy value, which includes defined, non-null values. This simplifies the code nicely!